trestle/core/markdown/control_markdown_node.py

```diff
@@ -462,6 +462,15 @@ def _read_parts(self, indent: int, ii: int, lines: List[str], parent_id: str,
                 name = 'objective' if id_.find('_obj') > 0 else 'item'
                 prop = common.Property(name='label', value=id_text)
                 part = common.Part(name=name, id=id_, prose=prose, props=[prop])
+                if id_ in [p.id for p in parts]:
+                    logger.error(
+                        f'Duplicate part id {id_} is found in markdown '
+                        f'{tree_context.control_id}. Please correct the part label in line {line}.'
+                    )
+                    raise TrestleError(
+                        f'Duplicate part id {id_} is found in markdown '
+                        f'{tree_context.control_id}. Please correct the part label in line {line}.'
```

Comment:
This duplication seems odd.  I checked other similar situations and I don't think this is common.  Is it redundant?

Reply:
Removed